crates/evm/src/trace/identifier/etherscan.rs

```diff
@@ -60,10 +60,14 @@ impl EtherscanIdentifier {
     /// Etherscan and compiles them locally, for usage in the debugger.
     pub async fn get_compiled_contracts(
         &self,
-    ) -> eyre::Result<(BTreeMap<ArtifactId, String>, BTreeMap<ArtifactId, ContractBytecodeSome>)>
-    {
+    ) -> eyre::Result<(
+        // TODO should use `ContractSources` but has circular import.
```

Comment:
should we resolve this todo?

Reply:
Yes, moved it to `crates/common/src/compile.rs` ! Thanks